Makefile

```diff
@@ -390,19 +390,19 @@ up: build ## Bring up local Grapl and detach to return control to tty
 
 .PHONY: down
 down: ## docker-compose down - both stops and removes the containers
-	$(WITH_LOCAL_GRAPL_ENV)
+	@$(WITH_LOCAL_GRAPL_ENV)
 	# This is only for killing the lambda containers that Localstack
 	# spins up in our network, but that docker-compose doesn't know
 	# about. This must be the network that is used in Localstack's
 	# LAMBDA_DOCKER_NETWORK environment variable.
 	$(MAKE) stop-nomad-detach
 	docker-compose $(EVERY_COMPOSE_FILE) down --timeout=0
-	docker-compose $(EVERY_COMPOSE_FILE) --project-name $(COMPOSE_PROJECT_INTEGRATION_TESTS) down --timeout=0
-	docker-compose $(EVERY_COMPOSE_FILE) --project-name $(COMPOSE_PROJECT_E2E_TESTS) down --timeout=0
+	@docker-compose $(EVERY_COMPOSE_FILE) --project-name $(COMPOSE_PROJECT_INTEGRATION_TESTS) down --timeout=0
```

Comment:
I'm somewhat curious if we even need these lines anymore. That said, we can deal with it in a different PR